app/frontend/stylesheets/config/_setup.scss

```diff
@@ -22,6 +22,8 @@ body {
   min-height: 100vh;
 }
 
+
+
 h1, h2, h3 {
   font-family: $headers-font;
 }
```

Comment:
There should only be one empty line between selectors. Remove line 25 and 26 please